src/components/views/dialogs/EndPollDialog.tsx

```diff
@@ -57,17 +56,10 @@ export default class EndPollDialog extends React.Component<IProps> {
         );
 
         if (endPoll) {
-            const endContent: IPollEndContent = {
-                [POLL_END_EVENT_TYPE.name]: {},
-                "m.relates_to": {
-                    "event_id": this.props.event.getId(),
-                    "rel_type": "m.reference",
-                },
-                [TEXT_NODE_TYPE.name]: message,
-            };
+            const closure = PollEndEvent.from(this.props.event.getId(), message).serialize();
```

Comment:
I don't understand the variable name `closure`.  Would suggest `endEvent` maybe?